lib/src/framework.dart

```diff
@@ -417,6 +422,21 @@ mixin HookElement on ComponentElement {
       _buildCache = super.build();
     } finally {
       _isOptionalRebuild = null;
+
+      _hooks.map((entry) {
```

Comment:
Should I apply this to the full list or only to `_currentHookState ` quite don't understand why only the first one is used